### PR TITLE
TX_INIT and TX_FINL redesign

### PR DESCRIPTION
**Pull request by OlivierBBB**

### CODE REVIEW

File: hub/generalities/revert/constants.tex [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+	\item $\finalizationEpsilon =  9$
+	\item $\selfdestructEpsilon = 10$
 \end{itemize}
-Two criteria have to be considered in setting these constants:
+Two criteria have to be considered in setting these constants.
+Let us define \col{max} as the maximum ``$\domStamp$''-offset encountered in constraints
 \begin{enumerate}
 	\item for any ``dom''-offset \col{d} used in a constraint to set $\domStamp \equiv \hubLambda \cdot \col{h} + \col{d}$ (where $\col{h} \equiv \hubStamp$) it should hold that $0 \leq \col{d} < \revertEpsilon$
-	\item $0 \leq \revertEpsilon < \selfdestructEpsilon < \hubLambda$
+	\item $\col{max} < \revertEpsilon < \finalizationEpsilon < \selfdestructEpsilon < \hubLambda$ where \col{max} is the maximum ``$\domStamp$''-offset encountered in constraints (i.e. the
```
Comment (letypequividelespoubelles) [inline hub/generalities/revert/constants.tex:13]: syntax: you defined 1 line before what's col{max}

**Comment by OlivierBBB** [comment]
@letypequividelespoubelles review comments addressed

**Review by letypequividelespoubelles — APPROVED** [review]
reviewed together :)

**Comment by Anon-im** [comment]
That is cool!

**Pull request merged**